LDAR_Sim/src/virtual_world/fugitive_emission.py

```diff
@@ -87,7 +87,7 @@ def check_if_repaired(self, emis_rep_info: EmisInfo) -> bool:
                 emis_rep_info.leaks_repaired += 1
                 emis_rep_info.repair_cost += self.get_repair_cost()
                 self._repair_date = self._start_date + timedelta(
-                    days=(self._active_days + self._days_active_b4_sim)
+                    days=int(self._active_days + self._days_active_b4_sim)
```

Comment:
Weird that this needed the typecast. These numbers should already be guaranteed to be integers